api/mock/facadeStub.go

```diff
@@ -40,6 +40,7 @@ type FacadeStub struct {
 	GetQueryHandlerCalled                       func(name string) (debug.QueryHandler, error)
 	GetValueForKeyCalled                        func(address string, key string, options api.AccountQueryOptions) (string, api.BlockInfo, error)
 	GetPeerInfoCalled                           func(pid string) ([]core.QueryP2PPeerInfo, error)
+	GetEpochStartDataForEpochCalled             func(epoch uint32) (*common.EpochStartDataAPI, error)
```

Comment:
Perhaps the value (not the pointer) can be returned here (optional).

Reply:
responded above